powersimdata/input/input_data.py

```diff
@@ -12,6 +13,77 @@
 profile_kind = {"demand", "hydro", "solar", "wind"}
 
 
+_file_extension = {
+    **{"ct": "pkl", "grid": "mat"},
+    **{k: "csv" for k in profile_kind},
+}
+
+
+BASE_URL = "https://bescienceswebsite.blob.core.windows.net/profiles"
+
+
+class InputHelper:
+    def __init__(self, data_access):
+        self.data_access = data_access
+
+    @staticmethod
+    def get_file_components(scenario_info, field_name):
+        ext = _file_extension[field_name]
+        file_name = scenario_info["id"] + "_" + field_name + "." + ext
+        from_dir = server_setup.INPUT_DIR
+        return file_name, from_dir
+
+    def download_file(self, file_name, from_dir):
+        self.data_access.copy_from(file_name, from_dir)
+
+
+class ProfileHelper:
+    @staticmethod
+    def get_file_components(scenario_info, field_name):
+        ext = _file_extension[field_name]
+        version = scenario_info["base_" + field_name]
+        file_name = field_name + "_" + version + "." + ext
+        grid_model = scenario_info["grid_model"]
+        from_dir = f"{server_setup.BASE_PROFILE_DIR}/{grid_model}"
```

Comment:
Is this a posixpath assumption?

Reply:
Or is this a URL, and the folder name matches the `server_setup` but we're not actually using the server?

Reply:
I would say that we don't need a BASE_PROFILE_DIR now that hat we fetch from the blob storage